### PR TITLE
prevent players from joining if instance has already started or done

### PR DESCRIPTION
**Pull request by carlnoval**
This may cause problems in the future if we're in the middle of demo and when someone from the crowd attempts to join.

### CODE REVIEW

**Comment by spioupiou** [comment]
I tried to join an ongoing instance and was successfully kicked out! :D  Thanks for the fix!
But there is a tiny typo in the other flash alert (I'll show the line in another comment) 

Screen Shot 2022-02-21 at 12 32 41

File: app/controllers/players_controller.rb [hunk 1]
```diff
@@ -6,6 +6,10 @@ def create
 
     if @instance.blank?
       redirect_to root_path, alert: "Lobby not found"
+    elsif @instance.status == 'ongoing'
+      redirect_to root_path, alert: "Game already started, try not to miss the next one."
+    elsif @instance.status == 'done'
+      redirect_to root_path, alert: "Game already finihsed, join the next one."
```
Comment (spioupiou) [inline app/controllers/players_controller.rb:12]: should be "finished" not "finihsed" 😆

Comment (carlncc) [reply app/controllers/players_controller.rb]: Thanks for the catch, apologies for I dunno how to type haha.

**Comment by spioupiou** [comment]
Thank you!!

**Pull request merged**